### ISSUE TITLE
Move [ERR] [ERROR] and similar to separate OpenType feature outside of `calt`

### ISSUE THREAD

**Issue by svipas** [fabrizioschiavi/pragmatapro#301]
When writing code in some cases it renders incorrectly, because ligatures like [ERR], [ERROR] are in `calt`. My suggestion is to move it to separate OpenType feature for e.g. `ss15`.

```ts
const [err] = someFn()
```

<img width="178" alt="image" src="https://github.com/user-attachments/assets/a5156cd5-899b-4a2f-b983-e1ee087904d4">

**Comment by fabrizioschiavi** [comment]
This makes sense. Thanks @svipas !

**Comment by matux** [comment]
Yes!

<img width="235" alt="Screenshot 2024-12-27 at 6 25 23 PM" src="https://github.com/user-attachments/assets/aaee4f83-5b5c-480d-9616-ae6a9693bae9" />

**Comment by wismill** [comment]
I would say migrate either all of them or none, for consistency.

**Comment by svipas** [comment]
Yeah I do agree with @wismill here. Better to migrate all of them to `ss15`.

**Comment by fabrizioschiavi** [comment]
Thank you all for pointing out the ideal solution!
Starting from PragmataPro version 0.9, these warning tags are disabled by default.
However, they can be enabled using the OpenType feature `ss15`

![Image](https://github.com/user-attachments/assets/a4d8a895-5d22-4798-9cd2-11192f2e4443)
